bayesian/api_v1.py

```diff
@@ -263,7 +263,16 @@ def get(ecosystem, package, version):
             )
             raise HTTPError(400, msg)
         if ecosystem == 'maven':
-            package = MavenCoordinates.normalize_str(package)
+            try:
+                package = MavenCoordinates.normalize_str(package)
+            except Exception:
+                msg = "Invalid maven format - {pkg}".format(
+                    pkg=package
+                )
+                metrics_payload.update({"status_code": 400, "value": time.time() - st})
+                _session.post(url=METRICS_SERVICE_URL + "/api/v1/prometheus", json=metrics_payload)
```

Comment:
This is kinda strange, but if it is implemented this way currently, then let's keep it.

Reply:
Well I agree with @msrb (and made similar comments elsewhere, for example there https://github.com/fabric8-analytics/fabric8-analytics-server/issues/550). The main problem is that the post does not specify timeout, so it will blocks the entire post queue when the Prometheus interface service won't be available for whatever reason.

4 @msrb: IIRC it does not call the real Prometheus, but a new service that are stateful and provides "new" funtionality like computation of moving average etc.